stat/int.go

```diff
@@ -0,0 +1,27 @@
+package stat // import "github.com/influxdata/influxdb/stat"
+
+import "sync"
+
+type Int struct {
+	mu    sync.RWMutex
```

Comment:
I'm not sure what the benchmark you're using is doing, but the operation itself is faster:

```
BenchmarkAtomicAdd-8    200000000                7.30 ns/op
BenchmarkMutexAdd-8     100000000               18.3 ns/op
```